source/features/remove-zero-packages.tsx

```diff
@@ -0,0 +1,24 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+function init(): void {
+	const packageElem: HTMLElement | null = select(`.numbers-summary a[href='/${getRepoURL()}/packages']`);
+	if (packageElem && packageElem.innerText.trim().startsWith('0')) {
+		const parentElem = packageElem.parentElement;
+		if (parentElem) {
```

Comment:
Is this ever possible to happen?

Reply:
:(
are you talking about the reference to the parent?